### DISCUSSION TITLE
Version 1.1.0

### DISCUSSION THREAD

**Discussion by oc007us** [JanusGraph/janusgraph#4680] [General]
Greetings,

I wanted to see when the version 1.1.0 is expected to be released. Seems that there have been a number of changes since the 1.0.0 release.

Thank you!
Oleg Cohen

**Comment by porunov** [comment]
1.1.0 is generally ready for the release other than one braking change related to queries optimization for the `drop` step.
However, we can't merge it until TinkerPop 3.7.3 is released. We tested it with the snapshot version of TinkerPop. 
As for ETA on TinkerPop 3.7.3 see the following reply in [discord](https://discord.com/channels/838910279550238720/838910279550238723/1270793395194826935).

**Reply by oc007us** [reply]
Thank you @porunov! Great to know and I will keep an eye on it.
